pkg/cmd/monitor_command/run_monitor_command.go

```diff
@@ -62,6 +63,9 @@ func NewRunMonitorCommand(ioStreams genericclioptions.IOStreams) *cobra.Command
 	cmd.Flags().StringVar(&monitorOpt.ArtifactDir,
 		"artifact-dir", monitorOpt.ArtifactDir,
 		"The directory where monitor events will be stored.")
+	cmd.Flags().StringVar(&monitorOpt.LoadBalancerType,
+		"lb-type", monitorOpt.LoadBalancerType,
+		"Set load balancer type, available options: internal-lb, service-network, external-lb (default)")
```

Comment:
question: is the default external-lb because a user can run it externally from the command line?

Reply:
Yes, correct